### PR TITLE
Fsf ga snap

### PR DESCRIPTION
**Pull request by Lukewh**
Add a google analytics event for the snap name created during fsf.

### CODE REVIEW

File: static/js/public/first-snap-flow.js [hunk 1]
```diff
@@ -44,60 +43,113 @@ function install(language) {
...
+      }
       if (initialCount === null) {
-        initialCount = count;
-      } else if (count !== initialCount) {
+        initialCount = data.count;
+        initialSnaps = data.snaps;
+      } else if (data.count !== initialCount) {
+        const newSnaps = newArrayValue(initialSnaps, data.snaps);
+
+        if (newSnaps.length > 0 && typeof ga !== "undefined") {
```
Comment (bartaz) [inline static/js/public/first-snap-flow.js]: Why is the same condition here as in the line 129? Seems like we send same ga event twice?

File: static/js/public/first-snap-flow.js [hunk 2]
```diff
@@ -1,39 +1,38 @@
-/* globals ClipboardJS */
+/* globals ClipboardJS, ga */
 
-import 'whatwg-fetch';
+import "whatwg-fetch";
```
Comment (bartaz) [inline static/js/public/first-snap-flow.js]: Aaaa, prettier changes again...

File: static/js/public/first-snap-flow.js (hunk 1 above)
Comment (Lukewh) [reply static/js/public/first-snap-flow.js]: Removed, was for my testing

File: static/js/public/first-snap-flow.js (hunk 2 above)
Comment (Lukewh) [reply static/js/public/first-snap-flow.js]: :+1: this is the only branch that should be affected, so once these changes are made, we should be good

File: static/js/public/first-snap-flow.js [hunk 3]
```diff
@@ -84,22 +84,61 @@ function getSnapCount(cb) {
   fetch("/snaps/api/snap-count")
     .then(r => r.json())
     .then(data => {
-      cb(data.count);
+      cb(data);
     });
 }
 
+function newArrayValue(arr1, arr2) {
```
Comment (bartaz) [inline static/js/public/first-snap-flow.js]: Not sure exactly if I read this function correctly, but could we name it more clearly to what it does? Something like `getArrayDiff` (if I get it right)?

Comment (Lukewh) [reply static/js/public/first-snap-flow.js]: :+1: I had a complete mind blank when naming this function, thanks. Will change it

**Pull request merged**